### ISSUE TITLE
Starting node with --kubeconfig still uses auto-detected master address

### ISSUE THREAD

**Issue by liggitt** [openshift/origin#887]
Run `openshift start node --kubeconfig=/openshift.local.certificates/admin/.kubeconfig`

Contents of .kubeconfig:
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority: root.crt
    server: https://10.245.2.2:8443
  name: master
contexts:
- context:
    cluster: master
    user: admin
  name: master-admin
current-context: master-admin
kind: Config
preferences: {}
users:
- name: admin
  user:
    client-certificate: cert.crt
    client-key: key.key
```

Node autodetects the local IP and uses it as the master address, ignoring the server addr in the .kubeconfig

**Comment by smarterclayton** [comment]
Node should not auto detect in the presence of kubeconfig.  Node should probably only auto detect when no arguments set.

**Comment by jwforres** [comment]
https://github.com/openshift/origin/pull/926

**Comment by jwforres** [comment]
Fixed in #926
